plusminus/plusminus.py

```diff
@@ -1032,8 +1075,7 @@ def evaluate(self):
         set_expression = pp.infixNotation(
             set_operand | var_name,
             [
-                (pp.oneOf("∩ &"), 2, pp.opAssoc.LEFT, SetBinaryOp),
-                (pp.oneOf("∪ |"), 2, pp.opAssoc.LEFT, SetBinaryOp),
+                (pp.oneOf("∩ & ∪ | \\ - − ∆"), 2, pp.opAssoc.LEFT, SetBinaryOp),
```

Comment:
I thought there was a precedence of operations for union and intersection, analogous to OR and AND, but the sources I found with Google indicate that there is not.